processor/src/main/resources/META-INF/services/javax.annotation.processing.Processor

```diff
@@ -0,0 +1 @@
+com.squareup.sqlbrite.SqlBriteProcessor
```

Comment:
Delete this file and add a dependency on [auto/service][1]. Then you can annotate the processor with `@AutoService(Processor.class)`.

 [1]: https://github.com/google/auto/tree/master/service